common/lib/xmodule/xmodule/library_content_module.py

```diff
@@ -434,6 +434,18 @@ def validate(self):
         validation = super(LibraryContentDescriptor, self).validate()
         if not isinstance(validation, StudioValidation):
             validation = StudioValidation.copy(validation)
+        library_tools = self.runtime.service(self, "library_tools")
+        if not (library_tools and library_tools.can_use_library_content(self)):
+            validation.set_summary(
+                StudioValidationMessage(
+                    StudioValidationMessage.ERROR,
+                    _(
+                        u"This course/system does not support content libraries. You probably "
```

Comment:
@bradenmacdonald You're the one who suggested this wording to me. Is there a reason 'system' needs to stay? I'll make the changes suggested by @catong here, but let me know if I should preserve 'system'.